src/CMakeLists.txt

```diff
@@ -359,11 +359,17 @@ if(USE_3DMAPPER)
   list(APPEND mudlet_HDRS glwidget.h)
 endif(USE_3DMAPPER)
 
+# credit to https://github.com/heavyai/heavydb/blob/master/CMakeLists.txt#L119
```

Comment:
This link will become outdated once the lines in the current blob get rearranged. You may want to specify a specific version, or remove the line number, to avoid linking an unrelated line after all.

Reply:
Good idea, done.